pkg/server/test/list_objects.go

```diff
@@ -194,9 +196,15 @@ func TestListObjectsRespectsMaxResults(t *testing.T, ds storage.OpenFGADatastore
 
 			ctx = typesystem.ContextWithTypesystem(ctx, typesystem.New(model))
 
+			checkCache := ccache.New(
+				ccache.Configure[*graph.ResolveCheckResponse]().MaxSize(100),
+			)
+
 			opts := []commands.ListObjectsQueryOption{
 				commands.WithListObjectsMaxResults(test.maxResults),
 				commands.WithListObjectsDeadline(test.listObjectsDeadline),
+				commands.WithCheckCache(checkCache),
+				commands.WithCheckQueryCacheTTL(1 * time.Millisecond),
```

Comment:
why do we need this change? the existing tests should still pass without it, since the caching is optional. Also note how the other options set here depend on each `test` :)

i think we are we missing new tests

Reply:
We don't - but I want to exercise having list object calls with cache.

Reply:
Added new test